src/app/branded/branded-checkout.component.js

```diff
@@ -41,6 +41,7 @@ class BrandedCheckoutController {
     }
     this.code = this.designationNumber
     this.tsysService.setDevice(this.tsysDevice)
```

Comment:
I suppose I could modify orderService and inject tsysService into it.

Reply:
But at the moment, I don't think tsysService is being passed to orderService.

Reply:
Which do you think is better? Importing another service or using local storage in another location?

Reply:
Loading from another service makes more sense to me, since it's already stored there, and the tsysService should probably be responsible for maintaining it.  I started by looking there, but the only methods I saw in it were `setDevice()` and `getManifest()`.  If I can just fetch the stored `device` property inside `tsysService`, that would be great, but when I saw that there was no official `getDevice()` method I wondered if it wasn't supposed to be accessible from there.
Would it be beneficial to use `sessionStorage` inside of `tsysService`, and store the `device` in `sessionStorage` instead of simply `this.device = device`?  I wonder if it would be more robust to do it that way.  Then `tsysService` can be the source of truth for the device id, and `orderService` can fetch from there.

Reply:
What benefit do hope to gain by putting it into session storage?

Reply:
I just looked up session storage vs. local storage.  My assumption was that it was a more secure or permanent way to store state data; that doesn't seem to be the case after all.  So I don't need to use session storage, but I think I would like to add a `getDevice()` method on `tsysService()` to have a formal way of accessing it.

Reply:
Right. Also need to fix whatever tests broke.